src/main/webapp/WEB-INF/pages/cartPage.jsp

```diff
@@ -0,0 +1,60 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<tags:master pageTitle="Product List">
+    <p>
+        Welcome to Expert-Soft training!
+    </p>
+    <p>
+        It is your cart!
+    </p>
+    <c:if test="${errors==null}">
+        <span class="successMess"><c:out value="${param.message}"/></span>
+    </c:if>
+    <table>
+        <thead>
+        <tr>
+            <td>Image</td>
+            <td>Description</td>
+            <td>Quantity</td>
+            <td>Price</td>
+        </tr>
+        </thead>
+        <c:set var="cart" value="${sessionScope['cartList'].list}"/>
+        <c:set var="error" value="${requestScope['errors']}"/>
+        <form id="deleteCartItem" action="${pageContext.servletContext.contextPath}/products/cart/deleteItem"
+              method="post">
+            <c:forEach items="${cart}" var="cartItem" varStatus="status">

```

Comment:
What if there are no items in the cart? Will we get table header with empty rows?
It's worth checking for empty cart before rendering the table.